internal/engine/dcwatch/event_watcher.go

```diff
@@ -26,12 +26,12 @@ func NewEventWatcher(dcc dockercompose.DockerComposeClient, docker docker.LocalC
 	}
 }
 
-func (w *EventWatcher) OnChange(ctx context.Context, st store.RStore, _ store.ChangeSummary) error {
-	if !w.watching {
+func (w *EventWatcher) OnChange(ctx context.Context, st store.RStore, summary store.ChangeSummary) error {
+	// TODO(nick): This should respond dynamically if the path changes.
+	if summary.IsLogOnly() || w.watching {
```

Comment:
This is the fix for the event watcher never running - I inadvertently flipped the condition back in #4364 🙀

Reply:
:grimacing: